Cargo.toml

```
@@ -99,6 +93,10 @@ version = "0.21"
 optional = true
 version = "0.18"
 
+[dependencies.xsalsa20poly1305]
+optional = true
+version = "0.4.2"
```

```suggestion
version = "0.4"
```
This may be better, unless the new crate is expected to introduce breaking changes in patch versions.

---

Normally I agree, but in writing this PR I actually added a few features it's using. See the [CHANGELOG.md](https://github.com/RustCrypto/AEADs/blob/master/xsalsa20poly1305/CHANGELOG.md).